### ISSUE TITLE
Add custom headers to ServerStream

### ISSUE THREAD

**Issue by JoseMontero001** [philmander/browser-bunyan#55]
hi, right now i am sending logs to a server using server-stream, but i need to add custom headers in the request, is there any way to do that ? here is an example of how i am doing it:

logInstance = createLogger({
          name: 'my-landing',
          streams: [{
            level: INFO,
            stream: new ServerStream({
              url: `${config.baseURLbff}logs`,
              method: 'POST',
            }),
          }],
        })

**Comment by dikaso** [comment]
We want to add Basic auth too.

**Comment by philmander** [comment]
No, but it wouldn't be hard to add it. If you want to try it yourself.

Accept custom header options here and add them to `this`:

https://github.com/philmander/browser-bunyan/blob/000a7211b459d91f4ee83220945ac3c31b18ea93/packages/server-stream/src/index.js#L15

 (these could include `Authorization` for basic auth; you would have to construct the auth header values and do the base64 encoding outside ServerStream)

Use the headers here:
https://github.com/philmander/browser-bunyan/blob/000a7211b459d91f4ee83220945ac3c31b18ea93/packages/server-stream/src/index.js#L37

and here:
https://github.com/philmander/browser-bunyan/blob/000a7211b459d91f4ee83220945ac3c31b18ea93/packages/server-stream/src/index.js#L69

**Comment by philmander** [comment]
I've added this feature in `v1.8.0`
